fastlane_bot/events/exchanges/solidly_v2.py

```diff
@@ -41,31 +49,70 @@ def add_pool(self, pool: Pool):
         self.pools[pool.state["address"]] = pool
 
     def get_abi(self):
-        return VELOCIMETER_V2_POOL_ABI if "velocimeter" in self.exchange_name else SOLIDLY_V2_POOL_ABI
+        if self.exchange_name == "velocimeter_v2":
+            return VELOCIMETER_V2_POOL_ABI
+        elif self.exchange_name == "equalizer_v2":
+            return EQUALIZER_V2_POOL_ABI
+        elif self.exchange_name == "scale_v2":
+            return SOLIDLY_V2_POOL_ABI
+        elif self.exchange_name == "aerodrome_v2":
+            return SOLIDLY_V2_POOL_ABI
+        elif self.exchange_name == "velodrome_v2":
+            return SOLIDLY_V2_POOL_ABI
+        elif self.exchange_name == "solidly_v2":
+            return SOLIDLY_V2_POOL_ABI
+        else:
+            raise self.WrongExchangeException(f"[exchanges/solidly_v2 get_Abi] Exchange {self.exchange_name} not in supported exchanges.")
```

Comment:
You can define `ABI_MAP` similar to `FEE_RESOLUTION_MAP`, and then turn this function into:
```
if self.exchange_name in ABI_MAP:
    return ABI_MAP[self.exchange_name]
raise self.WrongExchangeException(...)
```

Reply:
Updated